### PR TITLE
Implemented goBack Functionality

### PR DESCRIPTION
**Pull request by tharun208**
- Added GoBack Functionality
- Added tests for GoBack Function

### CODE REVIEW

File: src/api/Page.ts [hunk 1]
```diff
@@ -302,6 +302,9 @@ class Page extends EventEmitter {
       })
     `) as TResult
   }
+  async goBack(): Promise<string> {
```
Comment (deepsweet) [inline src/api/Page.ts]: I believe that it doesn't return anything, so it should be `Promise<void>`

File: src/api/Page.ts [hunk 2]
```diff
@@ -302,6 +302,9 @@ class Page extends EventEmitter {
...
+    return this._send('WebDriver:Back', {} ) as Promise<string>
```
Comment (deepsweet) [inline src/api/Page.ts]: which also means no `return` here, just `await`

**Comment by tharun208** [comment]
Fixed the changes !

**Comment by deepsweet** [comment]
thanks!

do you mind fixing lint issues? https://travis-ci.org/deepsweet/foxr/jobs/469398109

you can check it locally by running `yarn start lint`

**Comment by tharun208** [comment]
Hi,
    I fixed those linting changes

**Pull request merged**